pkg/trace/config/config.go

```diff
@@ -103,6 +103,9 @@ type AgentConfig struct {
 	// filtering
 	Ignore map[string][]string
 
+	// RequiredTags for filtering tags
```

Comment:
Is this comment really going to cut it? Will this help a reader understand what this field is?
```suggestion
	// RequiredTags holds a list of mandatory tags that each root span must hold for that trace to be considered.
```